public/assets/svg/github.tsx

```
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const IconGithub = ({ size = 24, color = '#fff', ...props }) => (
+  <svg
+    width={size}
+    height={size}
+    viewBox="0 0 98 96"
+    fill={color}
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)